tests/clickhouse-test

```diff
@@ -1934,22 +1945,44 @@ class TestSuite:
             tags = {tag.strip() for tag in tags}
             return tags
 
+        def parse_random_settings_limits_from_line(
+            line, comment_sign
+        ) -> Dict[str, Tuple[int, int]]:
+            if not line.startswith(comment_sign):
+                return {}
+            random_settings_limits_str = line[len(comment_sign) :].lstrip()  # noqa: ignore E203
+            random_settings_limits_prefix = "Random settings limits:"
+            if not random_settings_limits_str.startswith(random_settings_limits_prefix):
+                return {}
+            random_settings_limits_str = random_settings_limits_str[len(random_settings_limits_prefix) :]  # noqa: ignore E203
+            # limits are specified in a form 'setting1=(min, max), setting2=(min,max), ...'
+            random_settings_limits = re.findall(
+                "([^=, ]+) *= *(\([^=]+\))", random_settings_limits_str
+            )
+            random_settings_limits = {
+                pair[0]: make_tuple(pair[1]) for pair in random_settings_limits
+            }
```

Comment:
good to me, but maybe we can make a simpler-to-parse syntax (to avoid using regex) like 
`setting1=(min, max); setting2=(min,max)` : so that we can split by `;`, then split by `=`, then remove `()` and split by `,`.

Reply:
Sure, will change